app/uk/gov/hmrc/uknwauthcheckerapistub/utils/makers/OkMaker.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.uknwauthcheckerapistub.utils.makers
+
+import uk.gov.hmrc.uknwauthcheckerapistub.models.responses.EoriResults
+import uk.gov.hmrc.uknwauthcheckerapistub.utils.Constants
+
+class OkMaker {
+
+  def makeResults(eoris: Seq[String]): Seq[EoriResults] =
+    eoris.map { anEori =>
+      EoriResults(anEori, Constants.authorisedEoris.contains(anEori), 1)
```

Comment:
If Authorised = true, then it needs to be 0, otherwise 1 forthe last parameters.

Reply:
Do you mean like this?
![image](https://github.com/user-attachments/assets/1148a1b7-d26c-4fcd-aaa1-5b204c53a623)